### PR TITLE
Add a simple language selector. 

### PR DESCRIPTION
**Pull request by DevSplash**
https://github.com/mikebryant/ac-nh-turnip-prices/issues/136
The selector at the bottom of the page.

### CODE REVIEW

File: js/translations.js [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
+.use(i18nextXHRBackend)
+.use(i18nextBrowserLanguageDetector)
+.init({
+  fallbackLng: 'en',
+  debug: true,
+  backend: {
+    loadPath: 'locales/{{lng}}.json',
+  },
+}, (err, t) => {
+  const languages = [
```
Comment (promarcel) [inline js/translations.js:16]: Can you please directly add the following line: `['de', 'Deutsch'],`
Thank you very much!

Comment (DevSplash) [reply js/translations.js:16]: Done.

File: locales/zh-CN.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
     "decreasing": "递减型",
     "fluctuating": "波动型",
     "unknown": "不知道",
-    "large-spike": "大幅上涨（四期型）",
-    "small-spike": "小幅上涨（三期型）"
+    "large-spike": "大幅上涨（三期型）",
+    "small-spike": "小幅上涨（四期型）"
```
Comment (theRTC204) [inline locales/zh-CN.json:26]: I'm not fond of the pattern description in brackets here... `（三期型）`  and `（四期型）`, as it is not a direct translation of the text - it is additional context that is not provided to other languages. I would prefer to see this removed as there is additional conversation taking place in #41 about how to better communicate the behavior of the patterns.

Comment (DevSplash) [reply locales/zh-CN.json:26]: I added a demo image. 
As for `（三期型）` and `（四期型）`, you can find the reason they are so named [here](https://github.com/mikebryant/ac-nh-turnip-prices/pull/166). 
> (The "small-spike" is called "四期型" because the price would increase four (四) times in the week, and "large-spike" == "三期型" because the price would increase three (三) times in the week.)

But I have no idea whether it should be deleted. I thought @peter50216 could offer some ideas.

**Comment by theRTC204** [comment]
The selector could use some styling love, but we can raise that in another issue if that's outside your wheelhouse.

**Comment by DevSplash** [comment]
> The selector could use some styling love, but we can raise that in another issue if that's outside your wheelhouse.

OK, I’ll try it.

**Comment by promarcel** [comment]
From the UX side we should investigate as @theRTC204 mentioned.

I think we need to move the language selector also to the top of the page that people directly can see that they can switch the language.

**Comment by theRTC204** [comment]
I thought about suggesting moving it, but then since the app automatically changes to your system language, I felt being at the bottom was ok.

File: js/translations.js [hunk 3]
```diff
@@ -1,25 +1,46 @@
...
+    ['de', 'Deutsch'],
+    ['en', 'English'],
+    ['zh-CN', '简体中文'],
+    ['zh-TW', '繁體中文']
+  ],
```
Comment (mikebryant) [inline js/translations.js:21]: Should this be `;`?

File: locales/zh-CN.json [hunk 4]
```diff
@@ -22,8 +22,8 @@
     "decreasing": "递减型",
     "fluctuating": "波动型",
     "unknown": "不知道",
-    "large-spike": "大幅上涨（四期型）",
-    "small-spike": "小幅上涨（三期型）"
```
Comment (mikebryant) [inline locales/zh-CN.json:26]: This feels like an unrelated change - can it be removed?

Comment (mikebryant) [reply locales/zh-CN.json:26]: (Also other changes like this - I'd like to have this just be a PR about the language selector, not other things)

File: js/translations.js (hunk 3 above)
Comment (DevSplash) [reply js/translations.js:21]: The `languageSelector` is also defined, so it is correct. 
```suggestion
  const languages = [
    ['de', 'Deutsch'],
    ['en', 'English'],
    ['zh-CN', '简体中文'],
    ['zh-TW', '繁體中文']
  ],
  languageSelector = $('#language');
```

File: locales/zh-CN.json (hunk 4 above)
Comment (DevSplash) [reply locales/zh-CN.json:26]: I'll remove them and put up a new PR for them later.
